lib/house.rb

```diff
@@ -0,0 +1,47 @@
+class House
+  attr_reader :price, :address, :rooms
+
+  def initialize(price, address)
+    @price = price
+    @address = address
+    @rooms = []
+    @above_market_price = "$500,000"
```

Comment:
there's no reason for this to be an instance variable. you're never using this outside of the one `above_market_value?` method, so you can just check this value inside of that method.